dxr/plugins/python/analysis.py

```diff
@@ -50,17 +50,19 @@ def _analyze_file(self, path, encoding):
 
         """
         try:
-            syntax_tree = ast_parse(unicode_contents(path, encoding))
+            contents = unicode_contents(path, encoding)
+            syntax_tree = ast_parse(contents) if contents else None
```

Comment:
Consider whether this and the one in js.indexers should be `if contents is None`.